packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -1212,9 +1213,28 @@ function detachFiberMutation(fiber: Fiber) {
   fiber.return = null;
 }
 
-export function detachFiberAfterEffects(fiber: Fiber): void {
+export function detachFiberAfterEffects(
+  fiber: Fiber,
+  recurseIntoSibbling: ?boolean,
+): void {
   // Null out fields to improve GC for references that may be lingering (e.g. DevTools).
   // Note that we already cleared the return pointer in detachFiberMutation().
+  if (enableStrongMemoryCleanup) {
+    if (fiber.child) {
+      detachFiberAfterEffects(fiber.child, true);
+    }
+    if (fiber.sibling && recurseIntoSibbling === true) {
+      detachFiberAfterEffects(fiber.sibling, true);
+    }
+    if (fiber.stateNode) {
+      Object.keys(fiber.stateNode).forEach(key => {
+        if (key.indexOf('__react') === 0) {
+          delete fiber.stateNode[key];
```

Comment:
We already know the exact names, don't we? Can we use them directly?

https://github.com/facebook/react/blob/6d3ecb70dceb225af0cd990b46d6c44b852c1d82/packages/react-dom/src/client/ReactDOMComponentTree.js#L39-L44

Reply:
I added `unmountNode` to the HostConfig. Could use feedback if there's a better way.

Like I discussed with Brian this probably wont be necessary in the final version.